Tests/Tracking/Repository/IdentifyRepositoryTest.swift

```diff
@@ -147,4 +147,176 @@ class IdentifyRepositoryTest: UnitTest {
 
         waitForExpectations()
     }
+
+    // MARK: trackEvent
+
+    func test_trackEvent_expectCallHttpClientWithCorrectParamsNoBody() {
+        httpClientMock.requestClosure = { params, onComplete in
+            onComplete(Result.success(Data()))
+        }
+
+        let givenIdentifier = String.random
+
+        let expect = expectation(description: "Expect to complete")
+        expect.expectedFulfillmentCount = 2
+        integrationRepository
+            .addOrUpdateCustomer(identifier: givenIdentifier, body: EmptyRequestBody(), jsonEncoder: nil) { result in
+                XCTAssertEqual(self.integrationRepository.identifier, givenIdentifier)
+                expect.fulfill()
+            }
+
+        let givenEventName = String.random
+        let expectedBody = "{\"name\":\"\(givenEventName)\"}".data
+
+        httpClientMock.requestClosure = { params, onComplete in
+            guard case .trackCustomerEvent(let actualIdentifier) = params.endpoint else { return XCTFail() }
+
+            XCTAssertEqual(actualIdentifier, givenIdentifier)
+            XCTAssertEqual(params.body!, expectedBody)
+
+            onComplete(Result.success(params.body!))
+        }
+
+        integrationRepository.trackEvent(name: givenEventName, data: EmptyRequestBody(), timestamp: nil,
+                                         jsonEncoder: nil) { _ in
+            expect.fulfill()
+        }
+
+        waitForExpectations()
+
+        integrationRepository.removeCustomer()
```

Comment:
Is this for test cleanup? 

If so, you can put it in the test class function `tearDown()`.